docs/admins.rst

```diff
@@ -367,6 +367,31 @@ Example:
 .. _quay: https://quay.io/application/
 .. _`config.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/config.json
 
+Including OpenShift build annotations in Koji task output
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to include a ``build_annotations.json`` file in the task output
+of successful container image builds. This file may include any wanted
+OpenShift build annotations for the container build triggered by the Koji task
+in question.
+
+The ``koji-containerbuild`` plugin looks for a
+``koji_task_annotations_whitelist`` annotation in the OpenShift build
+annotations. This key should hold a list of annotations to be whitelisted for
+inclusion in the ``build_annotations.json`` file.
+
+If an empty ``build_annotations.json`` file would be generated through the
+process described above, the file is omitted from the task output.
+
+A list of annotations to be whitelisted in the
```

Comment:
To whitelist openshift annotations described above, you have to configure it in ``reactor_config_map``  ``koji`` section ``task_annotations_whitelist``. See  :ref:`config.yaml` for further reference.

Reply:
I was trying to explain that the reactor config map option will set the annotation that whitelists other annotations to be included in the JSON file. I tried to re-word the sentence. WDYT?

Reply:
ok